packages/web3-eth-personal/package.json

```diff
@@ -11,17 +11,25 @@
     "scripts": {
         "build": "rollup -c",
         "dev": "rollup -c -w",
-        "test": "jest"
+        "test": "jest",
+        "dtslint": "dtslint types --onlyTestTsNext"
     },
+    "types": "types",
     "dependencies": {
+        "typescript": "^3.3.0-dev.20181213",
```

Comment:
you don't need this as a dependency, typescript is a dependency on the root package.json